web/src/app/pages/main-page-container/main-page/map/map.component.html

```diff
@@ -18,28 +18,31 @@
   height="100%"
   width="100%"
   [options]="mapOptions"
-  (mapClick)="onMapClick($event)">
+  (mapClick)="onMapClick($event)"
+>
```

Comment:
Do we want these changes to be included? Was this an accident?

Reply:
It came from the linter.